packages/assets-controllers/src/TokenListController.ts

```diff
@@ -269,7 +275,6 @@ export class TokenListController extends StaticIntervalPollingController<TokenLi
   /**
    * This starts a new polling loop for any given chain. Under the hood it is deduping polls
    *
-   * @private
```

Comment:
Lint fixes.

This isn't needed and other controllers have also removed this JSDoc comment for their `_executePoll` methods